### PR TITLE
Gemelnet csv dump

### PR DESCRIPTION
**Pull request by thedod**
Dump a single month of a single kupa from gemelnet. (#3)
[Example spreadsheet](https://gist.github.com/anonymous/ee0b466abbb5a00c2572e4d068b94255).

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+pkg-resources==0.0.0
```
Comment (nirgn975) [inline requirements.txt]: I think it's come with setup tools. Do you even can install it like this?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# Open Pension Scraper
```
Comment (nirgn975) [inline README.md]: Re-roll from master and this should disappear (this isn't a change from this commit).

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+sudo: required
```
Comment (nirgn975) [inline .travis.yml:1]: Re-roll from master and this file should disappear (this file isn't a change from this commit).

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,163 @@
+data/*.csv
```
Comment (nirgn975) [inline .gitignore]: Re-roll from master and this file should disappear (this file isn't a change from this commit).

File: web_sources/gemelnet.py [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+        'dochot': 1, 'sug': 4,
+        'miTkfDivuach': period,
+        'adTkfDivuach': period,
+        'kupot': kupa_id
+        })
+    req.encoding = 'UTF-8'  # get() sometimes guesses wrong :/
+    return XML.fromstring(req.text)
+
+
+def save_csv_monthly_portfolio(kupa_id, year, month):
```
Comment (nirgn975) [inline web_sources/gemelnet.py:33]: Please add  a little docstring to the function.

File: web_sources/gemelnet.py [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+
+def parse_xml(xml):
+    return [
+        dict([(field.tag, field.text) for field in row])
+        for row in xml.iter('Row')]
+
+
+def load_xml_monthly_portfolio(kupa_id, year, month):
+    period = '{:04d}{:02d}'.format(year, month)
```
Comment (nirgn975) [inline web_sources/gemelnet.py:22]: Please add  a little docstring to the function.

File: web_sources/gemelnet.py [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+import csv
+import xml.etree.ElementTree as XML
+import requests
+
+API_URL = 'http://gemelnet.mof.gov.il/tsuot/ui/tsuotHodXML.aspx'
+OUTPUT_DIR = 'data'
+
+
+def parse_xml(xml):
+    return [
```
Comment (nirgn975) [inline web_sources/gemelnet.py:15]: Please add  a little docstring to the function.

File: web_sources/gemelnet.py [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+    pretty_month = '{:02d}/{:04d}'.format(month, year)
+    for r in result:
+        sheet.writerow([
+            kupa_id, pretty_month,
+            r['ID_NATUN'], r['SHM_NATUN'], r['ERECH_NATUN']])
+    return filepath
+
+
+if __name__ == '__main__':
+    import argparse
```
Comment (nirgn975) [inline web_sources/gemelnet.py]: `import` statement should be at the top of the file.

**Review by nirgn975 — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (thedod) [reply requirements.txt]: I have no idea where it came from, but things work well when I remove this line, so I will :wink:

File: .gitignore (hunk 4 above)
Comment (thedod) [reply .gitignore]: There's a small change: `data/*.csv` are created by `gemelnet.py` (added at this PR).

Comment (nirgn975) [reply .gitignore]: So just delete those changes, and add it to the end of the `.gitignore` file, under `### Custom ###` (if we don't have that section in the file, add it at the end).

File: README.md [hunk 9]
```diff
@@ -9,11 +9,23 @@ Open Pension is a "Hasadna" project, aiming to reveal the secrets behind the Isr
 
 ## Pre Requirements
 
-* Make sure you have Python 3.x and pip installed.
+* Make sure you have Python `3.x` and `virtualenv` installed.
 
 ## Installation
+```
```
Comment (nirgn975) [inline README.md]: Add shell to the code block, like:

``````
```shell
``````

**Review by nirgn975 — CHANGES_REQUESTED** [review]

**Review by nirgn975 — APPROVED** [review]

**Pull request merged**